src/quo2/components/markdown/list/style.cljs

```diff
@@ -0,0 +1,14 @@
+(ns quo2.components.markdown.list.style)
+
+(def container
+  {:flex-direction :row
+   :flex           1
+   :align-items    :flex-start
+   :margin-top     7})
```

Comment:
I checked the figma designs, but I couldn't find this `:margin-top`, probably a debug leftover?

Reply:
oof, yeah actually probably don;'t want this 🤔